### PR TITLE
Industry profile frontend

### PR DESCRIPTION
**Pull request by tejalreddy**
Created industry selector on profile with some minor other changes as well

### CODE REVIEW

File: client/components/Industry/IndustryButton.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    methods: {
+        async getIndustry() {
+            const r = await fetch(`/api/industry/users/${this.$store.state.user.id}`);
+            const res = await r.json();
+            if (!r.ok) {
+                throw new Error(res.error);
+            }
+            this.value = res.industry.industryType;
+        },
+        addIndustry(value) {
```
Comment (hxliu33) [inline client/components/Industry/IndustryButton.vue:60]: does a method containing an async function (`this.request()`) need to b async as well? i have no idea personally but j wondering if it worked when u tried it / if it makes a difference

File: client/components/Profile/PersonalInfoComponent.vue [hunk 2]
```diff
@@ -79,7 +84,7 @@ export default {
       editingInfo: false,
       fields: [
         {'name': 'graduationYear', 'display': 'Class of: '}, 
-        {'name': 'industry', 'display':  'Industry: '},
+        // {'name': 'industry', 'display':  'Industry: '},
```
Comment (hxliu33) [inline client/components/Profile/PersonalInfoComponent.vue:87]: why is this commented out / can we j remove it

File: client/components/common/BlockForm.vue [hunk 3]
```diff
@@ -112,6 +112,16 @@ export default {
           };
           const tags = await fetch('/api/tags', options);
           const tagsRes = await tags.json();
+
+          // delete industry when a user is deleted
+          // create an industry object when a new user is created
```
Comment (hxliu33) [inline client/components/common/BlockForm.vue:117]: remove this line ?? lol

File: server/user/collection.ts [hunk 4]
```diff
@@ -39,7 +39,12 @@ class UserCollection {
    * @return {Promise<Array<HydratedDocument<User>>>} - All the users
    */
   static async findAll(): Promise<Array<HydratedDocument<User>>> {
-    return UserModel.find({});
+    let newUsers:Promise<Array<HydratedDocument<User>>>;
+    const users = await UserModel.find({});
+    if (users) {
```
Comment (hxliu33) [inline server/user/collection.ts:44]: do you need to use the if statement here? i usually just populate whatever it returns like `return UserModel.find({}).populate('industry')`

**Review by hxliu33 — APPROVED** [review]
looks good to me as a whole, just have some minor things

File: client/components/Industry/IndustryButton.vue (hunk 1 above)
Comment (tejalreddy) [reply client/components/Industry/IndustryButton.vue:60]: I don't think it needs to be async

File: server/user/collection.ts (hunk 4 above)
Comment (tejalreddy) [reply server/user/collection.ts:44]: Tested it and if statement was not needed. Changed.

**Pull request merged**